pom.xml

```diff
@@ -1697,10 +1696,14 @@
                 <plugin>
                     <groupId>com.mycila</groupId>
                     <artifactId>license-maven-plugin</artifactId>
-                    <version>2.6</version>
+                    <version>2.8</version>
                     <configuration>
                         <skip>${skipVerifications}</skip>
-                        <header>${licensePath}</header>
+                        <header>com/mycila/maven/plugin/license/templates/CPAL.txt</header>
```

Comment:
you are right about EE. I've created a parallel PR https://github.com/mulesoft/mule-ee/pull/733